lib/util.js

```diff
@@ -1050,11 +1051,17 @@ module.exports = {
       encoding = contentObj[URLENCODED].encoding ? contentObj[URLENCODED].encoding : {};
       // create query parameters and add it to the request body object
       _.forOwn(bodyData, (value, key) => {
+
+        if (_.get(contentObj[URLENCODED], 'schema.type') === 'object') {
+          description = _.get(contentObj[URLENCODED].schema.properties[key], 'description', '');
```

Comment:
Why don't we need _.get for the .schema.props[key] part? This will throw an exception if `properties` doesn't exist.